CHANGES.md

```diff
@@ -8,6 +8,12 @@
   access the `GAP` module reliably, use the GAP variable `GAP_jl` instead.
 - Add the GAP function `JuliaType`,
```

Comment:
Actually probably best to keep all the "breaking" entries at the top, so move it above this line?